box-tool/src/main/java/smashdudes/core/boxtool/presentation/widgets/CharacterEditorWidget.java

```diff
@@ -82,28 +82,34 @@ private static void drawCharacterData()
         ImGui.sameLine();
         ImFloat jump = new ImFloat();
         jump.set(character.jumpStrength);
+        ImGui.pushID(1 + "");
         if(ImGui.inputFloat("", jump))
```

Comment:
Instead of using push/pop ID, use strings here for them. 
You can make the labels not appear by putting ## in front of them.

Imgui.inputFloat("##jumpInputID", jump)